test/stat_tracker_test.rb

```diff
@@ -31,4 +31,9 @@ def test_it_can_calculate_the_lowest_total_score
 
   end
 
+  def test_average_goals_per_game
+    # binding.pry
+    assert_equal 4.22, @stat_tracker.average_goals_per_game
```

Comment:
Your tests are phenomenal, I am super glad we have some of the expected results we need handy.  That helps us guide our work and I'm really appreciative that you found this work around.